packages/multi-select-combo-box/theme/lumo/vaadin-multi-select-combo-box-chip-styles.js

```diff
@@ -22,6 +22,10 @@ const chip = css`
     cursor: var(--lumo-clickable-cursor);
   }
 
+  :host(:is([readonly], [disabled]):not([part~='overflow'])) {
+    min-width: calc(var(--chip-min-width) - 1.25em);
```

Comment:
I guess it could be further adjusted by making the input not take any space when readonly:
From:
<img width="215" alt="Screenshot 2022-04-28 at 12 47 27" src="https://user-images.githubusercontent.com/1222264/165726732-3e157da2-6700-481f-ae74-6fd34d348104.png">

To:
<img width="212" alt="Screenshot 2022-04-28 at 12 49 59" src="https://user-images.githubusercontent.com/1222264/165726780-1bdab245-c190-43cf-b677-f29e55873da6.png">

Reply:
Thanks, that's a good idea. I will update the PR.

Reply:
@tomivirkki Updated logic and unit tests to check that chips are re-created on readonly / disabled change.

Reply:
nit: since we are using the value of `1.25em` that refers to the remove button size, my suggestion would be to extract this value into a private CSS property and use it here and on the `[part='remove-button']` definition, so if we ever come to change this size in the future, we may forget to change the value here as well.
The same applies for Material.